### PR TITLE
Dark theme support: restore and create wallet

### PR DESCRIPTION
**Pull request by SorinC6**
task: 
- https://emurgo.atlassian.net/browse/YOEXT-1316
- https://emurgo.atlassian.net/browse/YOEXT-1317

### CODE REVIEW

**Review by SorinC6 — COMMENTED** [review]
/check

File: packages/yoroi-extension/app/components/wallet/transactions/TransactionRevamp.js [hunk 1]
```diff
@@ -1057,7 +1056,40 @@ const icons = {
...
+    reward: { bg: ds.secondary_100, icon: ds.secondary_600 },
+    error: { bg: ds.magenta_100, icon: ds.magenta_500 },
+    stake: { bg: ds.secondary_100, icon: ds.secondary_600 },
+  };
+
+  return colorsByType[type] || { bg: ds.primary_100, icon: ds.primary_600 };
+};
+
+const IconWrapper = styled(Box)(({ isDark, colors, theme }) => ({
+  '& svg': {
```
Comment (Nebyt) [inline packages/yoroi-extension/app/components/wallet/transactions/TransactionRevamp.js]:
```suggestion
  '& svg': {
    width: '48px',
    height: '48px',
```

File: packages/yoroi-extension/app/components/wallet/transactions/TransactionRevamp.js [hunk 2]
```diff
@@ -1057,7 +1056,40 @@ const icons = {
...
+
 const TypeIcon = ({ type }) => {
   const Icon = icons[type];
-  return <Box sx={{ width: 40, height: 40 }}>{Icon && <Icon />}</Box>;
+  const theme = useTheme();
+  const colors = getColors(theme.palette.ds, type);
+  const isDark = theme.name === 'dark-theme';
+
+  return (
+    <Box sx={{ width: 40, height: 40 }}>
```
Comment (Nebyt) [inline packages/yoroi-extension/app/components/wallet/transactions/TransactionRevamp.js]:
```suggestion
    <Box sx={{ width: 48, height: 48 }}>
```

File: packages/yoroi-extension/app/components/wallet/summary/WalletSummaryRevamp.js [hunk 3]
```diff
@@ -224,21 +225,19 @@ export default class WalletSummaryRevamp extends Component<Props> {
             }}
           >
             <Grid item xs={4}>
-              <Typography component="div" variant="body2">
+              <Typography variant="body2" color="ds.text_gray_medium">
```
Comment (Nebyt) [inline packages/yoroi-extension/app/components/wallet/summary/WalletSummaryRevamp.js]:
```suggestion
              <Typography variant="body2" color="ds.text_gray_low">
```

File: packages/yoroi-extension/app/components/wallet/summary/WalletSummaryRevamp.js [hunk 4]
```diff
@@ -224,21 +225,19 @@ export default class WalletSummaryRevamp extends Component<Props> {
...
                 {intl.formatMessage({ id: 'wallet.summary.page.type' })}
               </Typography>
             </Grid>
             <Grid item xs={2} sx={{ textAlign: 'left' }}>
-              <Typography component="div" variant="body2">
+              <Typography variant="body2" color="ds.text_gray_medium">
```
Comment (Nebyt) [inline packages/yoroi-extension/app/components/wallet/summary/WalletSummaryRevamp.js]:
```suggestion
              <Typography variant="body2" color="ds.text_gray_low">
```

File: packages/yoroi-extension/app/components/wallet/summary/WalletSummaryRevamp.js [hunk 5]
```diff
@@ -224,21 +225,19 @@ export default class WalletSummaryRevamp extends Component<Props> {
...
                 {intl.formatMessage({ id: 'wallet.summary.page.status' })}
               </Typography>
             </Grid>
-            <Grid item xs={2} sx={{ textAlign: 'right' }}>
-              <Typography component="div" variant="body2">
-                {intl.formatMessage(globalMessages.feeLabel)}
-              </Typography>
+            <Grid item xs={2} sx={{ textAlign: 'right' }} color="ds.text_gray_medium">
```
Comment (Nebyt) [inline packages/yoroi-extension/app/components/wallet/summary/WalletSummaryRevamp.js]:
```suggestion
            <Grid item xs={2} sx={{ textAlign: 'right' }} color="ds.text_gray_low">
```

File: packages/yoroi-extension/app/components/wallet/summary/WalletSummaryRevamp.js [hunk 6]
```diff
@@ -224,21 +225,19 @@ export default class WalletSummaryRevamp extends Component<Props> {
...
+              <Typography variant="body2">{intl.formatMessage(globalMessages.feeLabel)}</Typography>
             </Grid>
-            <Grid item xs={4} sx={{ textAlign: 'right', pr: '30px' }}>
+            <Grid item xs={4} sx={{ textAlign: 'right', pr: '30px' }} color="ds.text_gray_medium">
```
Comment (Nebyt) [inline packages/yoroi-extension/app/components/wallet/summary/WalletSummaryRevamp.js]:
```suggestion
            <Grid item xs={4} sx={{ textAlign: 'right', pr: '30px' }} color="ds.text_gray_low">
```

File: packages/yoroi-extension/app/components/wallet/transactions/TransactionRevamp.js [hunk 7]
```diff
@@ -1057,7 +1056,40 @@ const icons = {
...
+    receive: { bg: ds.secondary_100, icon: ds.secondary_600 },
+    reward: { bg: ds.secondary_100, icon: ds.secondary_600 },
+    error: { bg: ds.magenta_100, icon: ds.magenta_500 },
+    stake: { bg: ds.secondary_100, icon: ds.secondary_600 },
+  };
+
+  return colorsByType[type] || { bg: ds.primary_100, icon: ds.primary_600 };
+};
+
+const IconWrapper = styled(Box)(({ isDark, colors, theme }) => ({
```
Comment (Nebyt) [inline packages/yoroi-extension/app/components/wallet/transactions/TransactionRevamp.js]:
```suggestion
const IconWrapper = styled(Box)(({ isDark, colors }) => ({
```

**Review by Nebyt — CHANGES_REQUESTED** [review]
@SorinC6 please check and accept suggestions.

**Review by SorinC6 — COMMENTED** [review]
/check

**Review by Nebyt — COMMENTED** [review]
/check

**Review by Nebyt — APPROVED** [review]

**Pull request merged**